docs/src/Combinatorics/simplicialcomplexes.md

```diff
@@ -0,0 +1,79 @@
+```@meta
+CurrentModule = Oscar
+```
+
+```@setup oscar
+using Oscar
+```
+
+```@contents
+Pages = ["simplicialcomplexes.md"]
+```
+
+# Simplicial Complexes
+
+## Introduction
+
+Abstract simplicial complexes provide a combinatorial way to define topological spaces.
+By no means every topological space arises in this way, but this is a (most) natural choice in a computational setup.
+
+A *simplicial complex* $K$ on a vertex set $V$ is a nonempty subset of $2^V$ such that for each $\sigma \in K$ and $\tau \subset\sigma$ we have $\tau\in K$.
+Here $V$ is usually $[n] = \{1,2,\dots,n\}$ for some $n\geq 0$.
+
+General textbooks offering details on the theory include:
+- [Koz08](@cite)
+
+## Construction
+
+```@docs
+SimplicialComplex(K::Vector{Vector{Int}})
+```
+
+### Surface examples
+
+```@docs
+torus()
+klein_bottle()
+real_projective_plane()
+```
+
+### Other examples
+
+```@docs
+complex_projective_plane()
+```
+
+## Basic properties
+
+```@docs
+nvertices(K::SimplicialComplex)
+dim(K::SimplicialComplex)
+f_vector(K::SimplicialComplex)
+h_vector(K::SimplicialComplex)
+euler_characteristic(K::SimplicialComplex)
+betti_numbers(K::SimplicialComplex)
+```
+
+## Connection to commutative algebra
+
+Let $K$ be a simplicial complex on $n$ vertices.
+The minimal non-faces of $K$ generate a square-free monomial ideal, known as the *Stanley-Reisner ideal* of $K$.
+The quotient of the polynomial ring (in $n$ variables, with integer coefficients) modulo that ideal is the *Stanley-Reisner ring*.
+For details see Chapter 5 of [BH09](@cite).
+```@docs
+minimal_nonfaces(K::SimplicialComplex)
+stanley_reisner_ideal(K::SimplicialComplex)
+stanley_reisner_ring(K::SimplicialComplex)
+stanley_reisner_ideal(R::FmpzMPolyRing, K::SimplicialComplex)
+stanley_reisner_ring(R::FmpzMPolyRing, K::SimplicialComplex)
+```
+
+## Saving and loading
+
+Objects of type `SimplicialComplex` can be saved to a file and loaded with the following two methods:
+```@docs
+save_simplicialcomplex(K::SimplicialComplex, filename::String)
```

Comment:
These names include the name of the type, which is neither necessary nor desirable. Better would be e.g. just `save`

Reply:
This is what I gad suggested myself, but @lkastner said that there had been objections.

I am much in favor of `save`.

Reply:
In the (stalled) PR #731, save and load are used. I don't see objections there. Would indeed be good to know what they are / who had them.